client/src/stories/ForgotPassword.stories.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import ForgotPassword from '../containers/ForgotPassword/ForgotPassword';
+import { Provider } from 'react-redux';
+import configureStore from '../store/configureStore';
+
+const store = configureStore();
+
+storiesOf('Forgot Password', module).add('Enter Email', () => {
```

Comment:
@Faria-Ejaz a good start, the idea behind story book is to show the different state of each component. So lets pick a component which has several states and document that.